asabiris/handlers/webhandler.py

```diff
@@ -176,11 +182,15 @@ async def render(self, request):
 		template_data = await request.json()
 
 		# Render a body
-		html = await self.render(template, template_data)
+		try:
+			html = await self.App.RenderReportOrchestrator.render(template, template_data)
+		except ValueError as e:
+			raise aiohttp.web.HTTPNotFound(text="{}".format(e))
+
 		# get pdf from html if present.
 		if fmt == 'pdf':
 			content_type = "application/pdf"
-			pdf = self.App.HtmlToPdfService.format(html)
+			pdf = self.App.PdfFormatterService.format(html)
```

Comment:
`self.App.PdfFormatterService.format_html()` ... to enable eventual other formats

Reply:
This comment is not resolved.

Reply:
@ateska  I thought it made sense to solve this when we introduce when we deal with other formats. Is that okay?

Reply:
hmms, ok then